tdp/core/models/operation_model.py

```diff
@@ -43,7 +47,9 @@ class OperationModel(BaseModel):
     start_time: Mapped[Optional[datetime]] = mapped_column(doc="Operation start time.")
     end_time: Mapped[Optional[datetime]] = mapped_column(doc="Operation end time.")
     state: Mapped[OperationStateEnum] = mapped_column(doc="Operation state.")
-    logs: Mapped[Optional[bytes]] = mapped_column(doc="Operation.")
+    logs: Mapped[Optional[bytes]] = mapped_column(
+        LargeBinary(LOGS_MAX_LENGTH), doc="Operation."
```

Comment:
We may want to take advantage of this PR to fix this doc

Reply:
done